src/main.js

```diff
@@ -120,6 +120,22 @@ if ('serviceWorker' in window.navigator) {
     });
 }
 
+window.setTimeout(function () {
```

Comment:
@humphd the formatting is a little off compared to the original, do you have any specific way to display this? or can I add a couple of touches?